ang/crmMonaco.ang.php

```diff
@@ -0,0 +1,22 @@
+<?php
+return array(
+  'ext' => 'civicrm',
+  'js' => [
+    'bower_components/monaco-editor/min/vs/loader.js',
+    'ang/crmMonaco.js',
+    //    'ang/crmMonaco/*.js',
+    //    'ang/crmMonaco/*/*.js',
+  ],
+  'css' => ['ang/crmMonaco.css'],
+  // 'partials' => ['ang/crmMonaco'],
+  'requires' => ['crmUi', 'crmUtil'],
+  'settings' => [
+    'paths' => [
+      'vs' => Civi::paths()->getUrl('[civicrm.bower]/monaco-editor/min/vs'),
+    ],
+  ],
```

Comment:
This is why the unit test fails @totten

Reply:
Nice to see the unit-test doing its job. :+1:

Fixed/squashed.